api/src/db/model/profile.ts

```diff
@@ -62,6 +83,21 @@ export default class ProfileModel extends Model {
         data.criticalSystem ? data.criticalSystem : false,
```

Comment:
@sbathgate Are you able to create a ticket to rename this in the db (schema) to `priority_system`?